bitcoin/src/consensus/params.rs

```diff
@@ -38,7 +38,7 @@ pub struct Params {
     /// Still, this should not affect consensus as the only place where the non-compact form of
     /// this is used in Bitcoin Core's consensus algorithm is in comparison and there are no
     /// compact-expressible values between Bitcoin Core's and the limit expressed here.
-    pub pow_limit: Work,
+    pub pow_limit: Target,
```

Comment:
This cannot be backported since it's a breaking change.

Reply:
My bad. Thanks for the lesson.